app/src/ui/about/about.tsx

```diff
@@ -269,6 +273,18 @@ export class About extends React.Component<IAboutProps, IAboutState> {
       return null
     }
 
+    if (isWindowsAndNoLongerSupportedByElectron()) {
+      return (
+        <DialogError>
+          This operating system is no longer supported. Software updates have
+          been disabled.{' '}
+          <LinkButton uri="https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/supported-operating-systems">
+            Support details
```

Comment:
```suggestion
            Supported Operating Systems
```

Reply:
Gosh, that's what I had originally but I tried so hard to not make it wordy 😞